.github/workflows/main.yml

```
@@ -0,0 +1,66 @@
+name: Lint code & Run tests
+
+on:
+  push:
+    branches:
+      - master
+      - develop
```

Can we write it like this?
```suggestion
      - [master, develop ]
```
However I am not sure :thinking:

---

Yup, it would work like this as well.

---

Done!